src/graphql/queries/leaderboard.js

```diff
@@ -0,0 +1,53 @@
+const { isArray, each } = require('lodash');
+const { lowercaseFilters, runPaginatedQuery } = require('./utils');
+
+const buildFilters = ({
+  userAddress,
+  eventAddress,
+} = {}) => {
+  if (!eventAddress && !userAddress) {
```

Comment:
this should be `||`? since GlobalLeaderboard can be queried by userAddress only?

Reply:
What i have right now means `if no eventAddress and no userAddress then error`, which is what we need right?

No matter which one, we need to pass at least one of them.

Reply:
gotcha, ya thats what we need.